docs/operator-manual/argocd-cmd-params-cm.yaml

```diff
@@ -177,6 +177,8 @@ data:
   applicationsetcontroller.enable.progressive.syncs: "false"
   # A list of glob patterns specifying where to look for ApplicationSet resources. (default is only the ns where the controller is installed)
   applicationsetcontroller.namespaces: "argocd,argocd-appsets-*"
+  # Enforce security by restricting the list of allowed scm providers (default "" is all scm providers)
```

Comment:
```suggestion
  # Enforce security by restricting the list of allowed SCM providers (default "" is all SCM providers). 
  # Setting this field is required when using ApplicationSets-in-any-namespace, to prevent users from
  # sending secrets from `tokenRef`s to disallowed `api` domains.
```